### ISSUE TITLE
Upper level perms needed on team invites accept/join

### ISSUE THREAD

**Issue by InnuendoSA** [Thutmose/ThutPerms#25]
I have a issue with team invites.
Players cant join a team if they are not part of the "mods" group.
If a invite is sent to a player it just says "You do not have an invite from Team ****" when the player try to join. It does show there is an invite under /teamInvites but still cant join.
Only after I move the player to the mods group, can the player join.
I have tried to do what you asked:
here is our last discussion on discord:

Thutmose - Today at 1:04 AM
@InnuendoSA I tested the stuff on my end, and when I did /editgroup default reset, people in the default group were able to invite to teams, so I suspect there is a permission needed

The group gets all the perms like the mods group if I reset it but still does not fix the problem.

**Comment by InnuendoSA** [comment]
![](https://cdn.discordapp.com/attachments/243868012329959424/454585933568933898/unknown.png)

![](https://cdn.discordapp.com/attachments/243868012329959424/454586036740161547/unknown.png)

**Comment by Thutmose** [comment]
I had only tested as far as getting the invite sent to the other player, as that was what you said was failing, I hadn't tried clicking on the link :D  

Try the version I sent in discord, it might have fixed the problem.

**Comment by Thutmose** [comment]
I should probably also note that this issue is on the wrong github page, it is an issue in thutessentials, not thutperms
